web/resources/views/next/base.blade.php

```diff
@@ -0,0 +1,172 @@
+        $social_rating
+
+        $title
+        $robots
+        $canonical
+        $license
+
+        $theme_color
+
+        $favicon_svg
+        $favicon_png
+        $favicon_apple
+        $favicon_mask
+
+    sections:
+        content
+
+    stacks:
+        preloads
+        styles
+        head
+        scripts
+- --}}
+
+<!DOCTYPE html>
+<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
+
+<head>
+    {{-- Functionality --}}
+    <meta charset="utf-8">
+    <base href="/">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+
+    {{-- Security --}}
+    <meta name="referrer" content="no-referrer">
+    <meta name="csrf-token" content="{{ csrf_token() }}" />
+    {{-- TODO:
+    <meta
+        http-equiv="Content-Security-Policy"
+        content=""
+    > --}}
+
+    {{-- Preloads, Preconnects --}}
+    {{-- TODO: figure out how to properly preload these:
+        <link rel="preload" href="{{ vite_asset('resources/css/base.scss') }}" as="style">
+        <link rel="preload" href="{{ vite_asset('resources/scripts/index.ts') }}"
+              as="script">
+        <link rel="preload"
+            href="{{ vite_asset('resources/fonts/variable/PublicSans-VariableFont.woff2') }}"
+            as="font" type="font/woff2" crossorigin>
+        <link rel="preload"
+            href="{{ vite_asset('resources/fonts/variable/Exo2-VariableFont.woff2') }}"
+            as="font" type="font/woff2" crossorigin>
+    - --}}
+    {{-- TODO: preload the user's locale file --}}
+    @stack('preloads')
+
+    {{-- OpenGraph --}}
+    {{-- TODO: verify that getLocale returns a well formed string for this --}}
+    <meta property="og:locale" content="{{ app()->getLocale() }}">
+    @isset($social_url)
+        <meta property="og:url" content="{{ $social_url }}">
+    @endisset
+    @isset($social_title)
+        <meta property="og:title" content="{{ $social_title }}">
+    @endisset
+    @isset($social_description)
+        <meta name="description" content="{{ $social_description }}">
+        <meta property="og:description" content="{{ $social_description }}">
+    @endisset
+    @isset($social_image)
+        <meta property="og:image" content="{{ vite_asset($social_image) }}">
+    @endisset
+    @isset($social_type)
+        <meta property="og:type" content="{{ $social_type }}">
+    @endisset
+
+    {{-- Twitter --}}
+    @isset($social_twitter_card)
+        <meta name="twitter:card" content="{{ $social_twitter_card }}">
+    @endisset
+    @isset($social_twitter_site)
+        <meta name="twitter:site" content="{{ $social_twitter_site }}">
+    @endisset
+    @isset($social_twitter_creator)
+        <meta name="twitter:creator" content="{{ $social_twitter_creator }}">
+    @endisset
+
+    {{-- Social Other --}}
+    @isset($rating)
+        <meta name="rating" content="{{ $rating }}">
+    @endisset
+
+    {{-- Browser Metadata --}}
+    @isset($title)
+        <title>{{ $title }}</title>
```

Comment:
Isn't a `<title>` required?

Reply:
Nope, at least technically.
![image](https://user-images.githubusercontent.com/34875062/136157166-8a999819-ffcd-4c2b-9803-6dfe8c193d5d.png)

For all practical purposes, yeah, you should _really_ give one. However, this layout isn't trying to validate what you give it, so, best to stick with the pattern and keep it optional.

Reply:
It isn't malformed if it doesn't include `<title>`. It's just bad practice. I don't see anything in MDN docs that make supplying a `<title>` element required.